qa/suites/fs/thrash/workloads/overrides/prefetch_dirfrags.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      mds:
+        mds oft prefetch dirfrags: false
```

Comment:
Suggest this patch:

````
diff --git a/qa/suites/fs/thrash/overrides/+ b/qa/suites/fs/thrash/overrides/%
similarity index 100%
rename from qa/suites/fs/thrash/overrides/+
rename to qa/suites/fs/thrash/overrides/%
diff --git a/qa/suites/fs/thrash/overrides/prefetch_dirfrags/no.yaml b/qa/suites/fs/thrash/overrides/prefetch_dirfrags/no.yaml
new file mode 100644
index 00000000000..91b45367934
--- /dev/null
+++ b/qa/suites/fs/thrash/overrides/prefetch_dirfrags/no.yaml
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      mds:
+        mds oft prefetch dirfrags: false
diff --git a/qa/suites/fs/thrash/overrides/prefetch_dirfrags/yes.yaml b/qa/suites/fs/thrash/overrides/prefetch_dirfrags/yes.yaml
new file mode 100644
index 00000000000..bd202f988c8
--- /dev/null
+++ b/qa/suites/fs/thrash/overrides/prefetch_dirfrags/yes.yaml
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      mds:
+        mds oft prefetch dirfrags: true
````

So we test both configurations.

Reply:
I applied it, thanks!